src/lib/util/anonymise.test.ts

```diff
@@ -0,0 +1,119 @@
+import { anonymise, anonymiseKeys } from './anonymise';
+
+const REGEX_MATCH = /^[a-f0-9]{9}@unleash\.run$/;
+
+test('anonymise', () => {
+    expect(anonymise('test')).toMatch(REGEX_MATCH);
+});
+
+test('anonymiseKeys', () => {
+    expect(anonymiseKeys(null, [])).toBe(null);
+    expect(anonymiseKeys(undefined, [])).toBe(undefined);
+    expect(anonymiseKeys(true, [])).toBe(true);
+    expect(anonymiseKeys(false, [])).toBe(false);
+    expect(anonymiseKeys('test', [])).toBe('test');
+    expect(anonymiseKeys(123, [])).toBe(123);
+    expect(anonymiseKeys([1, 2, 3], [])).toStrictEqual([1, 2, 3]);
+    expect(anonymiseKeys({ test: 'test' }, [])).toStrictEqual({ test: 'test' });
```

Comment:
I think here you could do something like:
```typescript
test.each([null, undefined, true, false, 'test', 123, [1, 2, 3], { test: 'test' })
    ('Without keys should return the same',
        (obj) -> { expect(anonymiseKeys(obj, []).toStrictEqual(obj); }
    );
```

Reply:
Tried addressing this in: https://github.com/Unleash/unleash/pull/3722/commits/e24dcaef1f0ffb087bc018d5ffa4837baec53cfd